src/ctapipe/calib/camera/calibrator.py

```diff
@@ -87,6 +88,16 @@ class CameraCalibrator(TelescopeComponent):
         ),
     ).tag(config=True)
 
+    process_flatfield_events = Bool(
```

Comment:
We already have a `EventTypeFilter` in the process tool for this, no new options should be needed.